src/main/java/seedu/foodrem/views/ItemView.java

```diff
@@ -35,16 +30,8 @@ public static Node from(Item item) {
         name.prefWidth(Double.MAX_VALUE);
 
         // Name and tags at the top left
-        final List<Node> tagsList = new ArrayList<>();
-        tagsList.add(new Label("Tags: "));
-        item.getTagSet().stream().sorted(Comparator.comparing(Tag::getName))
-                .forEach(tag -> tagsList.add(buildTagNodeFrom(tag.getName())));
-        if (tagsList.size() == 1) {
-            tagsList.add(new Label("-"));
-        }
-        final HBox tags = new HBox(tagsList.toArray(Node[]::new));
+        final HBox tags = new HBox(new Label("Tags: "), TagsView.from(item.getTagSet()));
         tags.setAlignment(Pos.CENTER_LEFT);
-        tags.setSpacing(SPACING_UNIT);
```

Comment:
We should still override the default spacing from the node that's returned for good practice even if they're the same value.